conf/messages

```diff
@@ -42,6 +43,7 @@ common.moreThanOnePensionScheme.individual = If more than one of your pension sc
 common.moreThanOnePensionScheme.agent = If more than one of your client’s pension schemes paid the tax, you can add these details later.
 common.pensionSchemeTaxReference = Pension Scheme Tax Reference
 
+common.error.amountMaxLimit = The total amount must be less than £100,000,000,000
```

Comment:
Not sure what copy should be used here. There are similar error messages which are not so general; for example: 

> The amount above the annual allowance must be less than £100,000,000,000

Reply:
So that we may wrap up this ticket, I've extracted this issue into it's [own](https://jira.tools.tax.service.gov.uk/browse/SASS-3126).